content/blog/2019/02/2019-02-08-let-s-celebrate-java-11-support.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Let's celebrate Java 11 Support on Jenkins"
+tags:
+- core
+- developer
+- java11
+- community
+- platform-sig
+author: alecharp
+links:
+  gitter: /jenkinsci/platform-sig
+  googlegroup: jenkins-platform-sig
+  sig: platform
+---
+
+NOTE: This is a joint blogpost prepared by the Java 11 Support Team: Adrien Lecharpentier, Baptiste Mathus, Kevin Earls, Mark Waite, Ramón León, Oleg Nenashev and María Isabel Vilacides.
+
+image:/images/logos/formal/256.png[Jenkins Java, role=center, float=right]
+
+We have worked hard for this and it's now here.
+We are happy to announce the **full support of Jenkins over Java 11**.
+This means you can now run your instance with a Java 11 JVM.
+
+Since June 2018, many events were organized to improve the code base of Jenkins to support running on Java 11.
+Outside of those events, Core and Plugins maintainers have worked hard to make sure they discover and solve as many potential issues related to Java 11 as possible.
+
+The effort to support Java 11 led to the creation of the https://github.com/jenkinsci/jep/blob/master/jep/211/README.adoc[JEP-211: Java 10+ support in Jenkins].
+It also formed the https://jenkins.io/sigs/platform[Platform Special Interest Group] to coordinate the Java 11 work and other platform support efforts.
+
+== Celebration
+
+We'd like to take a moment to thank everyone involved in these tasks: code contributors, issue reporters, testers, event planners and attendees and everyone in the community who has been involved in this.
+Thank you all!
```

Comment:
How about "and all those in the community who have generously lent their time and support to this effort"?